documentation/smg/images/08_HoracePR.svg

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Top left it says Herbertbranch.git

Reply:
Because the Herbert branch is pulled in as the Herbert component of Horace